package.json

```diff
@@ -64,6 +64,7 @@
     "@formatjs/intl-numberformat": "6.2.1",
     "@formatjs/intl-utils": "3.8.4",
     "@hookform/error-message": "0.0.5",
+    "@hugsmidjan/regulations-editor": "0.2.0",
```

Comment:
Shouldn't this be open source under the same terms as other Digital Iceland work? I couldn't find the source.

I assume it will be continuously updated, which doesn't fit great with our workflow and CI. Should we try to get it into the monorepo?

Reply:
No secrecy or closed-sourceness intended.

This code resides in a repo for a standalone tool we've already developed for cleaning up the texts of the old Regulation database. That tool has a very limited lifetime and it would thus not be productive try and bring that whole mess into the monorepo.

However, there are significant bits of code/logic that we **need** to code-share between that tool and the regulation-admin system, at the moment.

...and code-sharing seems easier in this direction, rather than trying to make the cleanup-tool somehow reach into the monorepo for the bits it needs.

My plan was to, once things have settled down a bit more, to split this module into its own public repo, and publish it under the @island.is npm scope.

Alternatively, we _could_ inline it in the monorepo when the time comes. I'm completely neutral on that, but I'd prefer to keep this code outside the monorepo for now.